app/controllers/admin/users_controller.rb

```diff
@@ -21,5 +22,72 @@ def unlock_account
       redirect_to user_search_path, notice: t("admins.user_search.account_unlocking_scheduled", name: u.username)
     end
 
+    def make_admin
+      unless Role.is_admin? @user.person
+        Role.add_admin @user.person
+        notice = "admins.user_search.add_admin"
+      else
+        notice = "admins.user_search.role_implemented"
+      end
+      redirect_to user_search_path, notice: t(notice, name: @user.username)
+    end
+
+    def remove_admin
+      if Role.is_admin? @user.person
+        Role.remove_admin @user.person
+        notice = "admins.user_search.delete_admin"
+      else
+        notice = "admins.user_search.role_implemented"
+      end
+      redirect_to user_search_path, notice: t(notice, name: @user.username)
+    end
+
+    def make_moderator
+      unless Role.moderator? @user.person
+        Role.add_moderator @user.person
+        notice = "admins.user_search.add_moderator"
+      else
+        notice = "admins.user_search.role_implemented"
+      end
+      redirect_to user_search_path, notice: t(notice, name: @user.username)
+    end
+
+    def remove_moderator
+      if Role.moderator? @user.person
+        Role.remove_moderator @user.person
+        notice = "admins.user_search.delete_moderator"
+      else
+        notice = "admins.user_search.role_implemented"
+      end
+      redirect_to user_search_path, notice: t(notice, name: @user.username)
+    end
+
+    def make_spotlight
+      unless Role.spotlight? @user.person
+        Role.add_spotlight @user.person
+        notice = "admins.user_search.add_spotlight"
+      else
+        notice = "admins.user_search.role_implemented"
+      end
+      redirect_to user_search_path, notice: t(notice, name: @user.username)
+    end
+
+    def remove_spotlight
+      if Role.spotlight? @user.person
+        Role.remove_spotlight @user.person
+        notice = "admins.user_search.delete_spotlight"
+      else
+        notice = "admins.user_search.role_implemented"
+      end
+      redirect_to user_search_path, notice: t(notice, name: @user.username)
+    end
+
+    private
+
+    def validate_user
+      @user = User.where(id: params[:id]).first
+      redirect_to user_search_path, notice: t("admins.user_search.does_not_exist") unless @user
+    end
+
```

Comment:
Layout/EmptyLinesAroundClassBody: Extra empty line detected at class body end.

Reply:
Nit: Please remove this line. :)